### DISCUSSION TITLE
Is it possible to define labels in SimpleSchema String allowedValues?

### DISCUSSION THREAD

**Discussion by rafaeltuelho** [vazco/uniforms#946] [Q&A] [answered]
Let's say I have a Select-String field using `SimpleSchema`

```json
        'policy.type': { 
          type: String,
          defaultValue: 'Select',
          allowedValues: ['COMPREHENSIVE', 'FIRE_THEFT', 'THIRD_PARTY'],
        },
```

How do I define friendly labels for each `allowedValues`?

**Comment by radekmie** [comment, marked as answer]
Hi @rafaeltuelho. There sure is! One is to use `options` prop that will be an array or object of the available options. The other would be to use the `transform` prop directly - it's a function transforming a value into its label. Both are described in the [`SelectField` props](https://uniforms.tools/docs/api-fields/#selectfield).

**Reply by rafaeltuelho** [reply]
Ah ok! Got it!
Something like:

```
policy: { type: Object, },
        'policy.type': { 
          type: String,
          defaultValue: 'Select',
          allowedValues: ['COMPREHENSIVE', 'FIRE_THEFT', 'THIRD_PARTY'],
          uniforms: {
            options:
              [
                { label: 'Comprehensive', value: 'COMPREHENSIVE' },
                { label: 'Fire Theft', value: 'FIRE_THEFT' },
                { label: '3rd Party', value: 'THIRD_PARTY' },
              ]
          }
        },
```

**Reply by rafaeltuelho** [reply]
actually, the trick is to define the `uniforms: {}` property in the `SimpleSchema` object :-). Maybe adding a small note in the docs would help close this gap.
